backend/services/AccountService.ts

```diff
@@ -116,9 +138,6 @@ export class AccountService {
           // END REMOVE:
           logger.debug(`Found account for referenceId:(${referenceId})`);
           return {
-            accessToken: createAuthToken(siwfData.accountId),
```

Comment:
Don't we still need an accessToken? Did this move somewhere else?

Reply:
I think this breaks the original sign in with frequency. I'm getting 401 errors after logging in now.

Reply:
I can commit a fix to revert the accessToken changes, let me know if there were other reasons to remove it.